cli/types/cypress.d.ts

```diff
@@ -2393,6 +2393,13 @@ declare namespace Cypress {
      * @default false
      */
     cmdKey: boolean
+    /**
+     * Where the caret should be placed when <input>, <textarea>, [contenteditable] elements are clicked.
+     * 'point' option can be only used with [contenteditable].
```

Comment:
we should add some logic to throw an error if caretPosition: 'point' is given to a non [contenteditable] element

Reply:
I added an error message.